mesonbuild/dependencies/base.py

```diff
@@ -956,16 +956,25 @@ def __init__(self, name, environment, kwargs, language=None):
             return
 
         modules = kwargs.get('modules', [])
+        cm_path = kwargs.get('module_path', [])
+        cm_args = kwargs.get('cmake_args', [])
         if not isinstance(modules, list):
             modules = [modules]
-        self._detect_dep(name, modules)
+        if not isinstance(cm_path, list):
+            cm_path = [cm_path]
+        if not isinstance(cm_args, list):
+            cm_args = [cm_args]
+        cm_path = [x if os.path.isabs(x) else os.path.join(environment.get_source_dir(), x) for x in cm_path]
+        if len(cm_path) > 0:
```

Comment:
just `if cm_path`. Using `len()` to test emptiness of contains is a python anti-pattern

Reply:
Fixed with the last commit.